doc/content/msr/msfr/griffin_pgh_model.md

```diff
@@ -334,22 +333,68 @@ symmetry and adiabatic boundaries.
 
 Legacy syntax for the energy equation is included [here](griffin_pgh_model_legacy.md).
 
+## Conservation of delayed neutron precursors
+
+The steady-state conservation of delayed neutron precursors (DNP) can be expressed as,
+\begin{equation}
+  \nabla \cdot (\vec{u}c_i) -  \nabla \cdot \epsilon_c \nabla c_i - \lambda_i c_i = \beta_i f,\quad i=1,\cdots,I,
+  \label{eq:dnp}
+\end{equation}
+where $c_i$, $\lambda_i$ and $\beta_i$ are the concentration, the decay constant and
+the production fraction per neutron generated from fission of the $i$-th group of
+delayed neutron precursor respectively. $f$ is the fission neutron production rate.
+$\epsilon_c$ is the eddy diffusivity for DNP.
```

Comment:
I do not know turbulent model, so this is my guess term added in DNP equation.

Reply:
Hi Yaqi - I'm pushing a clarification here.